cli/tests/integration/repl_tests.rs

```diff
@@ -122,6 +122,17 @@ fn pty_complete_primitives() {
   });
 }
 
+#[test]
+fn pty_complete_expression() {
+  util::with_pty(&["repl"], |mut console| {
+    console.write_text("Deno.\t\t");
+    console.write_text("y");
+    console.write_line("");
+    console.write_line("close();");
+    assert!(console.read_all_output().contains("Display all"));
```

Comment:
Could you actually assert that some of the expected fields in `Deno` namespace are in the output?